controller/gui/style.css

```diff
@@ -85,3 +81,14 @@ dl > dt {
 
     height: 90vh;
 }
+
+/* Network */
+
+.d-Network__ServiceDetail {
```

Comment:
Not sure whether to use BEM and start a mixed style here.

Since the project is rather small, I would either

- stick to the existing primitive style, or
- swap everything to BEM style for consistency.

Reply:
Opting to swap everything to BEM style for consistency. It will not be long because there are few styles. I’ll create another issue.